File: src/main/java/com/hjj/apiserver/repositroy/DepositRepositoryCustom.java
package com.hjj.apiserver.repositroy;

import com.hjj.apiserver.dto.DepositDto;
import com.querydsl.core.Tuple;

import java.util.List;

public interface DepositRepositoryCustom {

    List<DepositDto.ResponseDepositFindAll> findDepositAll();
    List<DepositDto.DepositIntrRateDesc>  findDepositByHome();
}

File: src/main/kotlin/com/hjj/apiserver/repository/deposit/DepositRepositoryCustom.kt
package com.hjj.apiserver.repository.deposit

import com.hjj.apiserver.dto.deposit.response.DepositFindAllResponse
import com.hjj.apiserver.dto.deposit.response.DepositIntrRateDescLimit10

interface DepositRepositoryCustom {
    fun findDepositAll(): List<DepositFindAllResponse>
    fun findDepositByHome(): List<DepositIntrRateDescLimit10>
}